aws/cloudformation/iam.yml.erb

```diff
@@ -730,20 +764,56 @@ roles.each do |name, config| %>
             Resource:
               - !Sub 'arn:aws:bedrock:${AWS::Region}:${AWS::AccountId}:knowledge-base/*'
 
-  CloudwatchDashboardViewerPolicy:
+  CloudwatchReaderPolicy:
     Type: AWS::IAM::ManagedPolicy
     Properties:
-      ManagedPolicyName: CloudwatchDashboardViewerPolicy
+      ManagedPolicyName: CloudwatchReaderPolicy
       PolicyDocument:
         Version: '2012-10-17'
         Statement:
+          - Effect: Allow
+            Action:
+              - cloudwatch:DescribeAlarms
+            Resource: 'arn:aws:cloudwatch:*:${AWS::AccountId}:alarm:*'
           - Effect: Allow
             Action:
               - cloudwatch:GetDashboard
-              - cloudwatch:ListDashboards
+            Resource: 'arn:aws:cloudwatch::${AWS::AccountId}:dashboard/*'
+          - Effect: Allow
+            Action:
+              - logs:DescribeLogStreams
+            Resource: 'arn:aws:logs:*:${AWS::AccountId}:log-group:*'
```

Comment:
I'd prefer to constrain this to just the specific Log Groups that we want to grant access to.